sql-mode.md

```diff
@@ -8,7 +8,7 @@ aliases: ['/docs/dev/sql-mode/','/docs/dev/reference/sql/sql-mode/']
 
 TiDB servers operate in different SQL modes and apply these modes differently for different clients. SQL mode defines the SQL syntaxes that TiDB supports and the type of data validation check to perform, as described below:
 
-After TiDB is started, modify `SET [ SESSION | GLOBAL ] sql_mode='modes'` to set SQL mode.
+After TiDB is started, you can use `SET [ SESSION | GLOBAL ] sql_mode='modes'` to set SQL mode.
 
 Ensure that you have `SUPER` privilege when setting SQL mode at `GLOBAL` level, and your setting at this level only affects the connections established afterwards. Changes to SQL mode at `SESSION` level only affect the current client.
 
```

Comment:
Please also update Line 15 in this file to match the Chinese changes at https://github.com/pingcap/docs-cn/pull/15883/files#diff-6ac3b6b6899c8c4dc81adb7f3a9d183dd650309ac829b98709cc3630949f47c4R15

Reply:
thanks